www/src/css/headline.css

```diff
@@ -1,22 +1,25 @@
 /** @define b-headline; weak */
 
 .b-headline {
+  margin-top:20px;
 }
 
-.b-headline .b-layout__container {
-  display:flex;
-  align-items: baseline;
+.b-headline__h2 {
+  margin:0 0 0 10px
 }
 
-.b-headline__h1 {
-  flex:1;
-  font-size:18px;
-  font-weight:400;
-  margin:20px 15px 5px 10px;
+.b-headline .b-headline__store-link {
```

Comment:
Is `b-headline__store-link` ever used ouside of `b-headline`? Does this selector need to be nested, as it seems that just the link class would work fine as well?